package-rfcx/rfcx/client.py

```diff
@@ -264,3 +265,23 @@ def streams(self, keyword=None, limit=1000, offset=0):
             List of streams"""
 
         return api_rfcx.streams(self.credentials.id_token, keyword, limit, offset)
+
+    def ingest_audio(self, stream_id, filepath, timestamp):
+        """ Ingest an audio to RFCx
+        Args:
+            stream_id: RFCx stream id
+            filepath: Local file path to be ingest
+            timestamp: Audio timestamp in iso format (with milliseconds)
+
+        Returns:
+            None.
+        """
+        regex = r'^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(\.[0-9]+)Z$'
+
+        iso_regex = re.compile(regex).match
+        if iso_regex(timestamp) == None:
+            print("Timestamp should be in iso format with milliseconds. For example: 2020-01-01T00:00:00.000Z")
+            return
```

Comment:
The ISO date format allows with and without milliseconds. It even allows with and without time I believe. Python has functions to parse iso dates which you can use.

https://stackoverflow.com/a/49784038/3301056